test/e2e/data/infrastructure-docker/main/clusterclass-quick-start.yaml

```diff
@@ -541,6 +541,7 @@ spec:
             ControlPlaneTemplate.machineTemplate.annotation: "ControlPlaneTemplate.machineTemplate.annotationValue"
         nodeDrainTimeout: 1s
       kubeadmConfigSpec:
+        useExperimentalRetryJoin: true
```

Comment:
what does the quickstart YAML actually do?
is it something exercised in * e2e tests.
perhaps there should be only a single e2e that uses this, but overall we should phase it out.

Reply:
Now gone, so resolving